### PR TITLE
point ideal position of bin

### PR DESCRIPTION
**Pull request by 708yamaguchi**
baxterの手先をbinの角に持ってくることで、人間がbinを正しい位置に配置できるようにしました。

### CODE REVIEW

**Comment by knorth55** [comment]
@708yamaguchi Please check https://github.com/708yamaguchi/jsk_apc/pull/1 and if you think it is ok, please merge it.

File: jsk_arc2017_baxter/euslisp/lib/point-shelf-position.l [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+       (coords-right (make-coords :pos (v+ #f(790 0 -185) #f(-300 -300 200)))))
+    (send *baxter* :rarm :inverse-kinematics coords-right :rotation-axis nil)
+    (send *baxter* :larm :inverse-kinematics coords-left :rotation-axis nil)
+    (send *ri* :angle-vector (send *baxter* :angle-vector) 5000 nil 0)
+    (send *ri* :wait-interpolation)))
+
+(warn "~% Commands ~%")
+(warn "First, you need to initialize *ri* and *baxter*, if you haven't done yet.~%")
+(warn "(jsk_arc2017_baxter::arc-init) : initialize *ri* and *baxter*~%")
+(warn "(point-shelf-position) : Baxter moves and points shelf position.~%")
```
Comment (wkentaro) [inline jsk_arc2017_baxter/euslisp/lib/point-shelf-position.l]: This is not lib, so should be placed in `euslisp` right? @pazeshun

Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/point-shelf-position.l]: Oh, that's right.
@708yamaguchi Please move this file

**Comment by knorth55** [comment]
+1
@wkentaro please merge this.

**Pull request merged**